internal/admin/ui/package.json

```diff
@@ -3,14 +3,27 @@
   "version": "0.0.0",
```

Comment:
@adith-a-danthi Let's add the node & npm version used here. 
Reference: -> https://stackoverflow.com/a/50798589/4257791